Plots/Satellites/NCL_sat_1.py

```diff
@@ -0,0 +1,162 @@
+
+    for x in np.arange(yradius, len(U.data), yradius*2): # X represents latitude
+        for y in np.arange(xradius, len(U.data[x]), xradius*2): # Y represents longitude
+
+            try:
+                # Fill a matrix of pressure values surrounding coordinate[x,y]
+                surroundingvals = U.data[x-xradius:x+xradius, y-yradius:y+yradius,]
+
+                if len(surroundingvals) == 2*xradius:
+
+                    for latindex in range(len(surroundingvals)):
+                        for lonindex in range(len(surroundingvals[latindex])):
+
+                            try:
+                                # Test every coordinate in surrounding area to see if it's 30 more than the center coordinate
+                                if surroundingvals[latindex][lonindex]-U.data[x][y]>pressureThreshold:
+                                    latval = lat[x]
+                                    lonval = lon[y]-180
+                                    print(U.data[x][y])
+                                    coordlist.append((latval, lonval))
+                                    break
+
+                            except Exception as E:
+                                continue
+
+            except Exception as E:
+                continue
+
+    return coordlist
+
+# Fix the artifact of not-shown-data around 0 and 360-degree longitudes
+wrap_U = gvutil.xr_add_cyclic_longitudes(U, "lon")
+
+###############################################################################
+# Create plot
+
+# Set figure size
+fig = plt.figure(figsize=(8, 8))
+
+# Set global axes with an orthographic projection
+proj = ccrs.Orthographic(central_longitude=270, central_latitude=45)
+ax = plt.axes(projection=proj)
+ax.set_global()
+
+# Add land, coastlines, and ocean features
+ax.add_feature(cfeature.LAND, facecolor='lightgray')
+ax.add_feature(cfeature.COASTLINE, linewidth=.5)
+ax.add_feature(cfeature.OCEAN, facecolor='lightcyan')
+ax.add_feature(cfeature.BORDERS, linewidth=.5)
+ax.add_feature(cfeature.LAKES, facecolor='lightcyan', edgecolor='k', linewidth=.5)
+
+# Make array of the contour levels that will be plotted
+contours = np.arange(948, 1060, 4)
+contours = np.append(contours, 975)
+contours = np.sort(contours)
+
+# Plot contour data
+p = wrap_U.plot.contour(ax=ax,
+                        transform=ccrs.PlateCarree(),
+                        linewidths=0.5,
+                        levels=contours,
+                        cmap='black',
+                        add_labels=False)
+
+# Specify array of contour levels to be labeled- These values were found by setting 'manual'
+# argument in ax.clabel call to 'True' and then hovering mouse over desired location of
+# countour label to find coordinate (which can be found in bottom left of figure window)
+
+# low pressure contour levels- these will be plotted as a subscript to an 'L' symbol
+#lowClevels = [(51.54, 169.59), (74.78, 4.54), (60.12, -57.0)]
+lowClevels = findLowPressureCoords()
```

Comment:
For this line, I would define your xradius, yradius, and pressureThreshold here instead of in the helper function. This way the function can be used in a more dynamic fashion for people that may use this code as a template for how to use/call functions in python, or want to look at different x/yradius or pressureThreshold values.